### ISSUE TITLE
Issue with references to StackOverflow

### ISSUE THREAD

**Issue by anoopelias** [danvk/dygraphs#684]
There are couple of places in the code base where there are references to code copied from StackOverflow.

https://github.com/danvk/dygraphs/blob/2cfded32d5da38aad8d9617654569f3e3f1c420d/src/dygraph-utils.js#L526

From what I understand, MIT license and copied code from SO [doesn't go together very well](http://meta.stackexchange.com/a/12537)

The guidance I have is to use SO for understanding and instead of copying an pasting, type it in yourself. That way we don't have to attribute it as well.

**Comment by tripleee** [comment]
There are changes coming: http://meta.stackexchange.com/questions/271080/the-mit-license-clarity-on-using-code-on-stack-overflow-and-stack-exchange
TLDR: New code on Stack Overflow will be MIT license.  The SO stance is that attribution should be provided if the author requests it.  It's not clear to me whether the link to the SO question in the code is sufficient; perhaps submit a clarifying question on MSO?

**Comment by tripleee** [comment]
I guess the comment should properly link to this specific answer? http://stackoverflow.com/a/384380/874188
